catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrCatalogProvider.java

```diff
@@ -106,7 +106,7 @@
 
     private SolrClient solr;
 
-    private SolrMetacardClient client;
+    private SolrMetacardClientImpl client;
```

Comment:
Initially, `SolrMetacardClient` was the impl and there was no interface. Eric added an interface to it to hide its functionality from `SolrCache` which pulls the client from the `SolrClientAdaptor`. But this class needs the impl.